lib/FormCustomProperties/tests/FormCustomPropertiesInteractor.js

```diff
@@ -36,17 +37,21 @@ import {
   label = text('[data-test-card-header-start]');
   note = text('[data-test-customproperty-note]');
   publicNote = text('[data-test-customproperty-public-note]');
-  value = text('[data-test-customproperty-value]');
+  value = value('[data-test-customproperty-value]');
   visibility = value('[data-test-customproperty-visibility]');
 
   nameField = scoped('[data-test-customproperty-name]', NameFieldInteractor);
   valueField = scoped('[data-test-customproperty-value]', ValueFieldInteractor);
   visibilityField = scoped('[data-test-customproperty-visibility]', visibilityFieldInteractor);
 
   fillNote = fillable('[data-test-customproperty-note]');
+  blurNote = blurrable('[data-test-customproperty-note]');
```

Comment:
The TextField interactor has a `fillAndBlur` interactor you might be able to use for the `fill*` interactors in place of `fillable`. That would let you avoid the `.fill().blur()` chaining you do in the tests.

Reply:
I tried it that way...seems it cant find the `input` that is being targetted because the `data-test-*` attributes are on the input? I might have to move the `data-test-*` atttribute onto a div/Fragment that wraps the new component I create above to target the value Field. Not sure if I can grab the Select and Note via the `data-test`, might have to tweak it.....will try